server/src/Domain/Entities/Flashcard.cs

```diff
@@ -1,9 +1,20 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using StudyZen.Domain.Constraints;
+
+
 namespace StudyZen.Domain.Entities;
 
 public sealed class Flashcard : BaseEntity
 {
     public int FlashcardSetId { get; set; }
```

Comment:
https://www.entityframeworktutorial.net/code-first/foreignkey-dataannotations-attribute-in-code-first.aspx

Reply:
Yeah, lets just add the collections. I think then some methods added in repositories are redundant, as you can load collections from related entities. E.g. FlashcardSet had a List of Flashcards, so just return them. You might run into this: https://learn.microsoft.com/en-us/ef/ef6/querying/related-data

Reply:
ok, gonna fix it with collections